dashboard/src/modules/components/OverviewComponent/NewRunsComponent.jsx

```diff
@@ -190,10 +195,10 @@ const NewRunsComponent = () => {
                       {rowActions ? (
                         <ActionsColumn
                           items={rowActions}
-                          isDisabled={
-                            item?.metadata[DATASET_OWNER] !==
-                            loginDetails?.username
-                          }
+                          // isDisabled={
+                          //   item?.metadata[DATASET_OWNER] !==
```

Comment:
Can we just remove this code instead of leaving it commented out?

Reply:
It is required. Once the user details is received in the response, these lines can be uncommented.

Reply:
I second Dave's suggestion:  if we _are_ going to retain commented-out code, we should have a `TODO` or other comment explaining why the code is commented out giving enough information to enable future developers to uncomment or delete it when appropriate.

But, here, I would be inclined to follow a different tack.  First, if Dave is right that all the table entries are already known to be owned by the user, perhaps we can just omit this code and not worry about it.

However, failing that (as I assume will be the case in the `SavedRunsComponent`), what happens if we leave this code uncommented?  Will it always cause the option to be disabled until the Server starts providing the owner username metadata?  If so, that seems like a good and reasonable approach:  as Dave says, that should allow you to do unit testing, and, then the code should "just work" once the Server is ready.

Of course, the other approach is for the Dashboard to skip trying to proactively disable the options, and for it to submit the delete/publish request to the Server, and then report the result -- success or failure -- back to the user, instead of trying to prevent the failure.

Reply:
As the table entries are of owned by the user, this check is not needed